### PR TITLE
Fix slug update on save

### PR DESCRIPTION
**Pull request by jrmi**
When we saved a scene, the slug was automatically updated with the name slugified. I remove this unwanted behaviour.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,62 @@
...
+```
+
+You can now edit your code. A django runserver is launched internally so the 
+this is an autoreload server.
+
+You can access to the api on http://localhost:8000/api/
+
+## Test
+
+To test the application, just launch:
```
Comment (nahuelange) [inline README.md]: «To run test suite» ?

File: terra_layer/views/layers.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ class SceneViewset(ModelViewSet):
     permission_classes = (ScenePermission,)
 
     def get_serializer_class(self,):
-        if self.action == "retrieve":
+        if self.action in ["retrieve", "update", "create"]:
```
Comment (DarkVisenya) [inline terra_layer/views/layers.py]: Why authorize these other actions?

File: test_terralayer/settings.py [hunk 3]
```diff
@@ -137,13 +137,17 @@
...
+            [-4.850, 46.776],
+            [-0.551, 48.886]
+        ],
     },
-    'second': {
-        'name': 'Second',
-        'pk': 2,
-    },
-}
+}
```
Comment (DarkVisenya) [inline test_terralayer/settings.py]: Missing new line

File: terra_layer/views/layers.py (hunk 2 above)
Comment (jrmi) [reply terra_layer/views/layers.py]: We want the detail serializer with the optionnal slug field for this to action.

File: test_terralayer/settings.py (hunk 3 above)
Comment (jrmi) [reply test_terralayer/settings.py]: Done

File: README.md (hunk 1 above)
Comment (jrmi) [reply README.md]: Done

File: test_terralayer/settings.py (hunk 3 above)
Comment (jrmi) [reply test_terralayer/settings.py]: done

**Pull request merged**